include/swift/Remote/MetadataReader.h

```diff
@@ -690,6 +690,28 @@ class MetadataReader {
                              RemoteAddress(StartOfValue));
   }
 
+  /// Given a known-opaque existential, discover if its value is inlined in
+  // the existential container.
```

Comment:
We usually use /// comments for the entire doxygen comment. I'm not sure if this is actually necessary, but at least it looks more consistent.